hack/util.sh

```diff
@@ -131,14 +131,14 @@ function start_os_server {
 	echo "[INFO] Using images:              ${USE_IMAGES}"
 	echo "[INFO] MasterIP is:               ${MASTER_ADDR}"
 
-	mkdir -p ${LOG_DIR}
+	mkdir -p "${LOG_DIR}"
 
 	echo "[INFO] Scan of OpenShift related processes already up via ps -ef	| grep openshift : "
```

Comment:
I'm not sure that the warning about `pgrep` over `ps | grep` is useful, so I blacklisted it.